### DISCUSSION TITLE
email Quote via API on create

### DISCUSSION THREAD

**Discussion by clhendy** [XeroAPI/xero-php-oauth2#321] [Q&A]
Hi,  I am trying to make the quote process from Xero be totally automated, but currently this isn't possible due to the requirement to log in and send the quote by email to the customer.  I can't find anything in the documentation for it.

I know that I can send the PDF separately once the quote has been made but it does not hold the link in for them to accept it.  I know that emailing invoices is possible so hoped that this could be introduced as well.

Is it possible to have the quote created via the API to be sent directly to the customer with the accept link embedded?

Many thanks

**Comment by rodjsta** [comment]
It appears you're correct.

Emailing an Invoice via the API also isn't the best as it won't include a PDF attachment.

Could you handle the emailing on your application side? Retrieve the Quote PDF from the API and then send the email with PDF attached from your system?

That is essentially what I did for the Invoices due to the PDF limitation.

**Reply by clhendy** [reply]
Hi, thanks for your reply

Yes that is what I am doing, which is OK, but when you send them from the actual XERO system it has an embedded link so that the customer can accept the quote directly.  And so far I have not found an efficient way to get the URL which Xero creates to be able to do this action.
